packages/teleport/src/components/RecoveryCodes/RecoveryCodes.tsx

```diff
@@ -40,16 +49,18 @@ export default function RecoveryCodesDialog({
     ).then(() => {
       selectElementContent(codesRef.current);
     });
+    captureRecoveryCodeEvent(CaptureEvent.PreUserRecoveryCodesCopyClickEvent);
   };
 
-  const handleContinue = () => {
-    if (username) {
-      userEventService.capturePreUserEvent({
-        event: CaptureEvent.PreUserRecoveryCodesContinueClickEvent,
-        username: username,
-      });
-    }
+  const onPrintClick = () => {
+    window.print();
+    captureRecoveryCodeEvent(CaptureEvent.PreUserRecoveryCodesPrintClickEvent);
+  };
```

Comment:
What's your thoughts on changing this to an event handler for `beforeprint`? That way we can capture events for people who use a keyboard shortcut to print the page.

Reply:
Resolved via DM; essentially:

> But, considering the question from the perspective of: "how does the user interact with our UI?" Then, capturing an event with a button click is the closest to the truth (including tab/enter methods). The event will open a dialog, and a user may print or click cancel, but they're not "interacting with our UI" at that time. Same if they use a shortcut browser or take a screenshot. That is why the naming is "...recoveryCodesPrint.click", not "...print."
Today, we're trying to see how a user works through the UI to onboard. We may find down the road that a ton of users who were given recovery codes are getting locked out. We can do additional event captures to dig into the issue if it is a problem area. Or add other requirements that the user clicks copy, etc., etc.